release/rotation

```diff
@@ -1,6 +1,8 @@
 # If you are adding someone to the rotation, remember to also give them permission
 # to access an ad-hoc machine for Windows testing. This can be done by opening a PR
 # like the following: https://github.com/DACH-NY/daml-language-ad-hoc/pull/37
+
```

Comment:
I'm not actually sure how the script will deal with a blank line. 🤔

Can you either check the script or remove that line? Thanks!

Reply:
Done, thanks!